src/Microsoft.AspNetCore.Authorization.Policy/Microsoft.AspNetCore.Authorization.Policy.csproj

```diff
@@ -14,6 +14,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Authentication.Abstractions" Version="$(MicrosoftAspNetCoreAuthenticationAbstractionsPackageVersion)" />
+    <PackageReference Include="Microsoft.AspNetCore.Routing" Version="$(MicrosoftAspNetCoreRoutingPackageVersion)" />
```

Comment:
Hmm, what's this for?

Reply:
The extension methods for adding authorization metadata to an endpoint - `RequireAuthorization` - use `IEndpointConventionBuilder` and `EndpointModel` which come from routing.

Reply:
We could move these builder types to `Microsoft.AspNetCore.Http.Abstractions`. They are pretty similar to `IApplicationBuilder` which lives there.

Reply:
We should probably do that.

Reply:
https://github.com/aspnet/HttpAbstractions/pull/1058